README.md

```diff
@@ -104,39 +104,91 @@ dropdown boxes.
 
 #### Selecting Machine Types
 
-The easiest way to get a list of all known Machine Types is with:
+The easiest way to get a list of all known Machine Type name is by utilizing either of the two APIs:
+
+#### Old API:
+
+```
+cli.get_machine_type_names(clean_strings_out)
+```
+
+#### New API:
 
 ```
-cli.get_machine_type_names()
+cli.get_machine_type_names(clean_strings_out=clean_strings_out)
+cli.get_machine_names(source_type=machine_type)
 ```
 
-This returns a python list with the machine type names.
+The function returns a Python list containing the names of machine types.
+
+An optional flag, `clean_strings_out`, determines the format of the returned list. If set to true, the list will use UI-based display names. If set to false, the list will contain Sight Machine internal machine types. If this flag is not provided, the function will default to returning UI-based display names.
```

Comment:
Add an example of the output with and without this string

Reply:
Done.